saleor/static/dashboard/scss/layouts/_orders.scss

```diff
@@ -5,3 +5,7 @@
     }
   }
 }
+
+.sku {
```

Comment:
In Bootstrap there is `text-muted` or something similar, don't we have it in Materialize? If not, it would be useful to have. I think we could use it more often then only with SKU.